tests/main/interfaces-libvirt/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Ensure that the libvirt interface works.
+
+systems: [ubuntu-16.04-64]
+
+summary: |
+    The libvirt interface allows a snap to access the libvirtd socket in order to manage
+    libvirt domains and other resources.
+
+    A snap which defines a libvirt plug must be shown in the interfaces list.
+    The plug must not be autoconnected on install and, as usual, must be able to be
+    reconnected.
+
+    A snap declaring a plug on this interface must be able to create and destroy a domain.
+    The test uses a snap that carries a unikernel built to be run on top of qemu, boot and
+    respond to ping. Once the domain is created, the test checks connectivity to the unikernel.
+
+prepare: |
+    echo "Given libvirt and qemu are installed"
+    sysctl -w net.ipv6.conf.all.disable_ipv6=1
+    trap "sysctl -w net.ipv6.conf.all.disable_ipv6=0" EXIT
+    apt install -y libvirt-bin qemu
+    adduser test libvirtd
```

Comment:
yes, in this mode, called with two non-option arguments, adduser will add the existing test user to the recently created libvirtd group, this is required for being able to interact with the socket later.